index/scorch/snapshot_rollback.go

```diff
@@ -149,10 +149,7 @@ func (s *Scorch) Rollback(to *RollbackPoint) error {
 
 		revert.snapshot = indexSnapshot
 		revert.applied = make(chan error)
-
-		if !s.unsafeBatch {
-			revert.persisted = make(chan error)
-		}
+		revert.persisted = make(chan error)
```

Comment:
should we also remove the conditional check on line 172 below?

Reply:
good point, will do